policyengine_us/parameters/gov/states/ar/tax/income/rates/main/rate.yaml

```diff
@@ -12,44 +12,64 @@ metadata:
     href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2022.pdf#page=1
   - title: 2021 Indexed Tax Brackets
     href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2021.pdf#page=1
-  - title: 2023 Indexed Tax Brackets
-    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2023.pdf#page=1
-    
+  - title: 2023-2024 Indexed Tax Brackets
+    href: https://casetext.com/statute/arkansas-code-of-1987/title-26-taxation/subtitle-5-state-taxes/chapter-51-income-taxes/subchapter-2-imposition-of-tax/section-26-51-201-individuals-trusts-and-estates
+  - title: 2014-2020 Indexed Tax Brackets
+    href: https://codes.findlaw.com/ar/title-26-taxation/ar-code-sect-26-51-201/
+
 brackets:
   - threshold:
       2021-01-01: 0
+      2023-01-01: 0
+      2024-01-01: 0
     amount:
       2021-01-01: 0
+      2023-01-01: 0
+      2024-01-01: 0
   - threshold:
       2021-01-01: 4_800
       2022-01-01: 5_100
-      2023-01-01: 5_300
+      2023-01-01: 5_100
+      2024-01-01: 5_100
     amount:
       2021-01-01: 0.02
+      2023-01-01: 0.02
+      2024-01-01: 0.02
```

Comment:
dont need values if they do not change, just the ones that do. 
if for example
2022-01-01: 10,000
then every year from then o (2023, 2024, ... , 2050) are all 10,000 unless we specify it other wise, so we only need to add values if they change